blueprints/addon/index.js

```diff
@@ -86,6 +86,19 @@ module.exports = {
     contents['ember-addon'] = contents['ember-addon'] || {};
     contents['ember-addon'].configPath = 'tests/dummy/config';
 
+    contents.files = [
+      'addon/**/!(.gitkeep)',
```

Comment:
I don't think we need a complicated regexp here, publishing a few (maybe just one?) empty file is not an issue. Lets remove these globs in favor of:

```suggestion
      'addon',
```

Reply:
Could I convince you otherwise? I'd prefer for it to not be worse than before. Depending on the addon, this would be three files: addon, app, and vendor.

Reply:
A few thoughts:

* they are three completely empty files and take up basically no weight
* we _probably_ shouldn't even be creating them to begin with 😛 
* we should lint for `.gitkeep` files in directories with other contents

Reply:
I can agree with the second two points. Is that enough reason to start publishing these files? Could we iterate and have these globs temporarily?

Reply:
> Could we iterate and have these globs temporarily?

We could, but I find the extra blueprint churn worse than the current state...

Reply:
Dropping this concern, let's move forward...